market_nextdoor_api/urls.py

```diff
@@ -17,10 +17,21 @@
 from django.contrib import admin
 from django.urls import path
 from .views import market_views, customer_views, vendor_views, item_views, customer_views, preorder_views, weather_views, preorder2_vendors_views, preorder2_customers_views
+from .views.v2 import (
```

Comment:
might have mentioned this, and this really just naming thing, but do we need to rename all the files with v2 infront? the distinction of the folder is good enough. Anywho really just a small thing and matter of preference, so disregard if you disagree.

Reply:
yeah this is one of those things that while it works, it may not be the best format for versioning, and I think we'll just have to choose, how we want to version and go all in from there.